packages/blade/src/components/List/getOrderedListItemBullet.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Converts number to roman number
+ *
+ */
+const romanize = (number: number): string => {
+  const romanKeys: Record<string, number> = {
+    m: 1000,
+    cm: 900,
+    d: 500,
+    cd: 400,
+    c: 100,
+    xc: 90,
+    l: 50,
+    xl: 40,
+    x: 10,
+    ix: 9,
+    v: 5,
+    iv: 4,
+    i: 1,
+  };
+  let result = '';
+
+  Object.keys(romanKeys).forEach((key) => {
+    result += key.repeat(Math.floor(number / romanKeys[key]));
+    number %= romanKeys[key];
```

Comment:
can't this just work with a map from 1-i, 2-ii and so on? why does it needs this logic?

Reply:
This just makes it scalable and the logic won't fail even if we add 100+ items.  Same reasoning for a,b,c... level